composer.json

```diff
@@ -20,7 +20,8 @@
         "kriswallsmith/buzz": "~0.15"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "~4.0",
+        "symfony/phpunit-bridge": "^2.7.4"
```

Comment:
I don't keep it. It's just PR separation.

I'll rebase it once when #59 will be merged.